src/main/java/org/opensearch/knn/index/codec/KNN80Codec/KNN80DocValuesConsumer.java

```diff
@@ -248,7 +260,7 @@ private void createKNNIndexFromScratch(FieldInfo fieldInfo, KNNCodecUtil.Pair pa
                 KNNConstants.INDEX_DESCRIPTION_PARAMETER,
                 FAISS_BINARY_INDEX_DESCRIPTION_PREFIX + parameters.get(KNNConstants.INDEX_DESCRIPTION_PARAMETER).toString()
             );
-            parameters.put(KNNConstants.VECTOR_DATA_TYPE_FIELD, VectorDataType.BINARY.getValue());
+            IndexUtil.updateVectorDataTypeToParameters(parameters, VectorDataType.BINARY);
```

Comment:
Same here. By having this method, I need to go inside the method to see what it actually does which is just adding binary value in the parameters.

Reply:
Answered above